tests/unit/api/apiUtils/quotas/quotaUtils.js

```diff
@@ -500,12 +500,40 @@ describe('processBytesToWrite', () => {
     it('should return content length of the bject metadata is the action is a copy (getObject authz)', () => {
         bucket.isVersioningEnabled.returns(true);
         objMD = { 'content-length': 100 };
-        contentLength = 150;
 
         const bytes = processBytesToWrite('objectCopy', bucket, versionId, 0, objMD);
 
         assert.strictEqual(bytes, 100);
     });
+
+    it('should return content length of the bject metadata is the action is a copy part (getObject authz)', () => {
```

Comment:
`should return content length of the bject metadata **if** the action is a copy part (getObject authz)`